docs/login.rst

```diff
@@ -245,10 +244,17 @@ Then you want to enable ID Site in your express configuration::
       }
     }
 
-When ID Site is enabled, any request for ``/login`` or ``/register`` will cause a
-redirect to ID Site.  When the user is finished at ID Site they will be
-redirected to `/idSiteResult` on your application.  Our library will handle
-this request, and then redirect the user to the ``nextUri``.
-
+When ID Site is enabled, any request to ``/login`` or ``/register`` will cause a
+redirect to ID Site.  When the user is finished at ID Site, the user will be
+redirected to the ``/idSiteResult`` endpoint on your application with a secure JWT
+passed as a URL parameter. The job of ``/idSiteResult`` is to validate this JWT
+and return the Stormpath Account. After this is done, the authenticated user
```

Comment:
It doesn't return the account directly, I'd like us to cover: it exchanges the jwtResult for an access and refresh token, which are stored in cookies during the redirect to the `nextUri`